src/js/quizapiWrapper.js

```diff
@@ -0,0 +1,69 @@
+import fetch from "node-fetch";
+export class QuizapiWrapper {
+    constructor(category) {
+        this.apiKey = "EAco9tkerMBqitin1irWhCDipOxYHxJ5A5xe1nqQ"
+        this.data;
+        this.checkCategory(category) 
+        this.url = `https://quizapi.io/api/v1/questions/?apiKey=${this.apiKey}&tags=${this.category}`;
+    };
+
+    async getData() {
+        
+        const callApi = (url) => {
+            const data = fetch(url)
+            .then(res => res.json())
+            .then(json => json)
+            .catch(err => console.error('Error:', err));
+            return data;
+        }
+        
+        this.data = await callApi(this.url);
+        var allQuestions = [];
```

Comment:
We should use ES6 javascript. `var` should not be used nowadays. Instead use `const` or `let`.